File: js/programmers/level.1/prgm_42840_1.js
function solution(answers) {
  const one = [1, 2, 3, 4, 5];
  const two = [2, 1, 2, 3, 2, 4, 2, 5];
  const three = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
  let total = [0, 0, 0];
  let answer = [];

  for (let i = 0; i < answers.length; i++) {
    if (one[i % one.length] === answers[i]) total[0]++;
    if (two[i % two.length] === answers[i]) total[1]++;
    if (three[i % three.length] === answers[i]) total[2]++;
  }

  const max = Math.max(...total);

  for (let i = 0; i < total.length; i++) {
    if (total[i] === max) answer.push(i + 1);
  }

  return answer;
}

File: python/programmers/level.1/prgm_42840_1.py
def solution(answers):
    first = [1, 2, 3, 4, 5]
    second = [2, 1, 2, 3, 2, 4, 2, 5]
    third = [3, 3, 1, 1, 2, 2, 4, 4 ,5, 5]
    total = [0, 0, 0]
    answer = []

    for i in range(len(answers)):
        if (first[i % len(first)] == answers[i]): total[0] += 1
        if (second[i % len(second)] == answers[i]): total[1] += 1
        if (third[i % len(third)] == answers[i]): total[2] += 1

    for i in range(len(total)):
        if (total[i] == max(total)): answer.append(i+1)

    return answer